integration/fixtures/bastion/main.tf

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
`integration` as a directory name is not clear what kind of integration is being referred to.  It would be awesome and much more clear, if this directory is actually named `integration_tests` as in the doc when we considered the options.

Reply:
Renamed.